src/css/custom.css

```diff
@@ -26,6 +26,7 @@
   --ifm-color-primary-lighter: #1e9124;
   --ifm-color-primary-lightest: #22a428;
   --ifm-code-font-size: 95%;
+  --ifm-link-decoration: underline;
```

Comment:
We need to either make the color bright or make the text underline as per here.

<img width="533" alt="image" src="https://github.com/microsoft/playwright.dev/assets/17984549/fff6fc6a-68d0-4427-8e3c-6b1ee3ae3b80">

Making it more bright is not possible without keeping the contrast ration as per here:

<img width="235" alt="image" src="https://github.com/microsoft/playwright.dev/assets/17984549/af4dceb7-5b31-46d9-936c-9b0d7cfd4873">

So maybe just underline by default on light theme?

Reply:
I don't like the look when all links are underlined, feels much heavier than without it:

<img width="613" alt="image" src="https://github.com/microsoft/playwright.dev/assets/9798949/1d054c7a-8d78-4b0e-93d8-67479d25218b">

Without:
<img width="644" alt="image" src="https://github.com/microsoft/playwright.dev/assets/9798949/1b4b6c4b-af35-4122-9b6b-11275e044ff4">

Reply:
Yes we will revert it, the approval was for everything except for that.